lab_7_llm/main.py

```diff
@@ -32,6 +28,7 @@ class RawDataImporter(AbstractRawDataImporter):
     """
     A class that imports the HuggingFace dataset.
     """
+    _raw_data: DataFrame
```

Comment:
This parameter is already created in the class `AbstractRawDataImporter`

Reply:
I added this line here because MyPy checks do not pass due to not specifying the Dataframe of `raw_data`